index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div id="main">
+      <h1 id="title">Muhmmd and Mahao</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt=""
```

Comment:
- [ ] Great job in the other two comments you asked for changes in a friendly and polite way and explained why does he need the changes in this comment you did great too but you didn't mention the right reason for the change you asked so Please edit this comment to be like: 
"Kindly add a proper value to the `alt` attribute for each image you add because it will be better semantically that way and gives your site better accessability".

Reply:
@Omar-Muhamad 
I have already made it more clear as you mentioned, could you please check it before making a submission.